src/sentry/static/sentry/app/components/charts/chartZoom.jsx

```diff
@@ -214,14 +214,14 @@ class ChartZoom extends React.Component {
       return children(props);
     }
 
-    const dateFormat = useShortDate ? 'MMM Do' : 'lll';
+    const dateFormat = useShortDate ? 'MMM Do' : 'll[\n]LT';
     const hasShortInterval = useShortInterval(this.props);
     const xAxisOptions = {
       axisLabel: {
         formatter: (value, index) => {
           const firstItem = index === 0;
           const format = hasShortInterval && !firstItem ? 'LT' : dateFormat;
-          return getFormattedDate(value, format, {local: !utc});
+          return `${getFormattedDate(value, format, {local: !utc})}`;
```

Comment:
Why do you need to string interpolate here?

Reply:
stray string template.